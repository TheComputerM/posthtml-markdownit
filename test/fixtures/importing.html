<md src="test/fixtures/test.md">
  # Imported content should be above me
</md>
