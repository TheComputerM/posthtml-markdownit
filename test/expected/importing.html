<h1>Hello there</h1>
<h2>Imported content should be above me</h2>
